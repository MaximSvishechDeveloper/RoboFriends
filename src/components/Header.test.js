import { shallow } from "enzyme";
import Header from "./Header";
import React from "react";

it("renders without crashing", () => {
  expect(shallow(<Header />)).toMatchSnapshot();
});
