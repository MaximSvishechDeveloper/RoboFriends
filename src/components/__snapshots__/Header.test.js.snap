// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<h1
  className="f1"
>
  RoboFriends
</h1>
`;
